provisioning/roles/wazuh/ansible-filebeat-oss/defaults/main.yml

```diff
@@ -1,7 +1,11 @@
 ---
 filebeat_version: 7.10.2
 
+<<<<<<< HEAD
+wazuh_template_branch: "{{ SOURCE_REFERENCE }}"
+=======
 wazuh_template_branch: 4.9.0
+>>>>>>> 5a460cbb266f502fd2376326ab69cf7ab35759da
 
```

Comment:
Bad conflict resolution

Reply:
Done in [3624d76](https://github.com/wazuh/wazuh-qa/pull/5552/commits/3624d7627e9dc6d1fd82ace2ed034ded3b32fb32)